tests/cases/fourslash/autoImportPackageJsonImportsPattern.ts

```diff
@@ -0,0 +1,18 @@
+/// <reference path="fourslash.ts" />
+
+// @module: nodenext
+
+// @Filename: /package.json
+//// {
+////   "imports": {
+////     "#*": "./src/*"
+////   }
+//// }
+
+// @Filename: /src/something.ts
+//// export function something(name: string): any;
+
+// @Filename: /a.ts
+//// something/**/
+
+verify.importFixModuleSpecifiers("", ["#something.js"]);
```

Comment:
@andrewbranch should this autocomplete `#something.js`? I'm investigating some stuff and I concluded the conclusion that perhaps this should actually be just `#something`. `@module: nodenext` doesn't prescribe the specifier ending - and since this `package.json` doesn't have a type it's an implied CJS file so the ending is optional.

note for myself: fixing this would likely depend on using `getModuleSpecifierEndingPreference` appropriately